templates/collaboration.html

```diff
@@ -24,37 +24,82 @@
 }
 .button{
 	padding: 10px;
+	border:1px solid black;
+	border-radius: 3px;
 }
 .collaborator-section{
+	margin: 10px;
 	display: flex;
 	justify-content: space-between;
 }
 .input{
-	width: 100%;
+	width: 90%;
+	border:1px solid #ddd;
+	border-radius: 3px;
 }
 .space{
 	padding: 10px;
 }
+
+.user-photo{
+	padding-right: 10px;
+	width: 50px;
+	height: 50px;
+}
+
+.collaborator{
+	display: flex;
+	padding: 10px;
+	padding-left: 20px;
+	border-bottom: 1px solid #ddd;
+}
+
+.collaborator-info{
+	padding: 40px;
+	border-bottom: 1px solid #ddd;
+	text-align: center;
+}
+.search-text{
+	padding: 10px;
+	font-weight: bold;
+	font-size: 20px;
+}
+.collaborator .username a{
+	text-decoration: none;
+	color: black;
+}
+
 </style>
 {{template "nav.html" .}}
+{{if .Authorized}}
 <div class="main-section">
 	<div class="heading">
 		<span>Collaborators</span>
 		<span>Push access to the repository</span>
 	</div>
-	<div class="info">
+	{{if .IsCollaboratorsAvailable}}
+		{{range .Collaborators}}
+		<div class="collaborator">
+			<div class="user-photo"><img class="user-photo" src="{{.UserImage}}"></div>
+			<div class="username"><a href="#">{{.Username}}</a></div>
+		</div>	
+	{{end}}
```

Comment:
This end is for range block. Indentation is wrong.

Reply:
DONE